content/zh/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -17,42 +17,42 @@
 
 		<div class="row">
 			<div class="col-md-8">
-    		<h3>Objectives</h3>
+    		<h3>目的</h3>
 				<ul>
-					<li>Learn about a Service in Kubernetes</li>
-					<li>Understand how labels and LabelSelector objects relate to a Service</li>
-					<li>Expose an application outside a Kubernetes cluster using a Service</li>
+					<li>了解Kubernetes中的服务</li>
```

Comment:
@jierYang  pls   sign cla
```suggestion
					<li>了解 Kubernetes 中的服务</li>
```
中英文之间保持一个空格
/retest
/check-cla